build/css/style.css

```diff
@@ -593,8 +625,16 @@ video {
   display: none;
 }
 
-.flex-1 {
-  flex: 1 1 0%;
+.w-\[340px\] {
+  width: 340px;
```

Comment:
Hello @UmutesiMMA  and @gsngdavid kindly change this width. It's making the lines not be responsive. you can either make it a percentage or `rem` to adjust as the screen decreases. 

Below I will show you a screenshot that the above `px` has caused.